deploy/gcp/tidbclusters.tf

```diff
@@ -26,7 +21,7 @@ module "default-tidb-cluster" {
   gke_cluster_name           = module.tidb-operator.gke_cluster_name
   cluster_name               = var.default_tidb_cluster_name
   cluster_version            = var.tidb_version
-  kubeconfig_path            = module.tidb-operator.kubeconfig_path
+  kubeconfig_path            = data.local_file.kubeconfig.filename
```

Comment:
in helm provider, and this module we can reference data source `local_file.kubeconfig` which presents kubeconfig local file data source in root module.